Jenkinsfile

```diff
@@ -1 +1,5 @@
-buildPlugin(configurations: buildPlugin.recommendedConfigurations())
+buildPlugin(configurations: [
+  [ platform: 'linux', jdk: '8', jenkins: '2.222.4' ],
+  [ platform: 'linux', jdk: '11', jenkins: '2.222.4', javaLevel: '8' ],
+  [ platform: 'windows', jdk: '8', jenkins: '2.222.4' ],
```

Comment:
You shouldn't use a named version of Jenkins. using `jenkins: null` would use the version specified in `pom.xml`.